catalog/transformer/catalog-transformer-html/src/main/resources/templates/template.hbs

```diff
@@ -0,0 +1,136 @@
+<html>
```

Comment:
Should hbs files have the copyright header?

Reply:
This brings up an interesting question. Should the copyright header be at the top of each template I have or should it only be in the base template file?

If I attach the copyright header to each template then it will show up all over an HTML export which would muddy it up quite a bit.

Reply:
I think it only needs to appear in source code.

Reply:
I think it should be on all of them. IIRC handlebar comments don't get included in the produced html

Reply:
Sounds good. I added it into each template